build.sbt

```diff
@@ -32,11 +30,9 @@ val parserComb        = "org.scala-lang.modules" %% "scala-parser-combinators"
 val postgresql        = "org.postgresql"         %  "postgresql"                  % postgresqlVer
 val restAssured       = "io.rest-assured"        %  "rest-assured"                % restAssuredVer
 val restAssuredScala  = "io.rest-assured"        %  "scala-support"               % restAssuredVer
-val scalaMock         = "org.scalamock"          %% "scalamock-scalatest-support" % scalaMockVer
+val scalaMock         = "org.scalamock"          %% "scalamock"                   % scalaMockVer
 val scalaTest         = "org.scalatest"          %% "scalatest"                   % scalaTestVer
 val slick             = "com.typesafe.slick"     %% "slick"                       % slickVer
-val sprayJson         = "io.spray"               %% "spray-json"                  % sprayJsonVer
```

Comment:
Am I correct in thinking this is not used anywhere and thus can be removed?

Reply:
No, you're not correct. Spray is used in [HyperionJsonProtocol.scala](https://github.com/mthmulders/hyperion/blob/master/app/src/main/scala/hyperion/rest/HyperionJsonProtocol.scala) as well as [AppInfoService.scala](https://github.com/mthmulders/hyperion/blob/master/app/src/main/scala/hyperion/rest/AppInfoService.scala).

Could you please put it back?

Reply:
Since you asked nicely. The dependency wasn't actually added to the project, though. So I'll do that as well